src/test/scala-2.11/markets/orders/orderings/bid/BidPriceTimeOrderingSpec.scala

```diff
@@ -15,40 +15,30 @@ limitations under the License.
 */
 package markets.orders.orderings.bid
 
-
-import akka.actor.ActorSystem
-import akka.testkit.TestKit
-
 import java.util.UUID
 
 import markets.orders.BidOrder
 import markets.orders.limit.LimitBidOrder
 import markets.orders.market.MarketBidOrder
 import markets.tradables.Tradable
-import org.scalatest.{BeforeAndAfterAll, FeatureSpecLike, GivenWhenThen, Matchers}
+import org.scalatest.{BeforeAndAfterAll, FeatureSpec, GivenWhenThen, Matchers}
 
 import scala.collection.immutable
 import scala.util.Random
 
 
-class BidPriceTimeOrderingSpec extends TestKit(ActorSystem("BidPriceTimeOrderingSpec")) with
-  FeatureSpecLike with
-  GivenWhenThen with
-  Matchers with
-  BeforeAndAfterAll {
-
-  /** Shutdown actor system when finished. */
-  override def afterAll(): Unit = {
-    system.terminate()
-  }
-
+class BidPriceTimeOrderingSpec extends FeatureSpec
+  with GivenWhenThen
+  with Matchers
+  with BeforeAndAfterAll {
+  
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Whitespace at end of line](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/2707644711/issues/source?bid=2921361&fileBranchId=3343521#l34)